src/Address.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Address.h"
+
+#include "Bitcoin/Address.h"
+#include "Bitcoin/CashAddress.h"
+#include "Bitcoin/Bech32Address.h"
+#include "Ethereum/Address.h"
+#include "Icon/Address.h"
+#include "Nimiq/Address.h"
+#include "Ripple/Address.h"
+#include "Tendermint/Address.h"
+#include "Zcash/TAddress.h"
+
+#include <TrustWalletCore/TWHRP.h>
+
+using namespace TW;
+
+std::string TW::loadAddress(const Data& data, TWCoinType coin) {
+    switch (coin) {
+    case TWCoinTypeBinance:
+        return Tendermint::Address(HRP_BINANCE, data).encode();
+
+    case TWCoinTypeBitcoin:
+        return Bitcoin::Bech32Address(HRP_BITCOIN, 0, data).encode();
+
+    case TWCoinTypeBitcoinCash:
+        return Bitcoin::CashAddress(data).string();
+
+    case TWCoinTypeEthereum:
+    case TWCoinTypeEthereumClassic:
+    case TWCoinTypeVeChain:
+    case TWCoinTypeWanChain:
+        return Ethereum::Address(data).string();
+
+    case TWCoinTypeICON:
+        return Icon::Address(data, TWIconAddressTypeAddress).string();
+
```

Comment:
`Icon::Address(data, TWIconAddressTypeAddress).string();`
`Bitcoin::Bech32Address(HRP_LITECOIN, 0, data).encode();`

is it possible to make it consistent? so it's always `string()` ?

Reply:
ok